lib/tasks/remove_audio_asset_derivatives.rake

```diff
@@ -0,0 +1,20 @@
+namespace :scihist do
+  desc """
+    Goes through all the derivatives and if they are audio, removes them.
```

Comment:
To be accurate, if they are marked 'webm' or 'small_mp3', it doesn't check to see if their content-type is audio, which it could. It doens't need to just saying the description is inaccurate. 

Should we also note in comments "can/should be removed once it's run in production and no longer needed."

Reply:
Good catch. I wrote the comment before the code :) Will amend.